.github/workflows/e2e-test-tune-api.yaml

```diff
@@ -0,0 +1,37 @@
+name: E2E Test with tune API
+
+on:
+  pull_request:
+    paths-ignore:
+      - "pkg/ui/v1beta1/frontend/**"
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  e2e:
+    runs-on: ubuntu-22.04
+    timeout-minutes: 120
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Setup Test Env
+        uses: ./.github/workflows/template-setup-e2e-test
+        with:
+          kubernetes-version: ${{ matrix.kubernetes-version }}
+      
+      - name: Run e2e test with tune API
+        uses: ./.github/workflows/template-e2e-test
+        with:
+          experiments: ""
+          # Comma Delimited
+          trial-images: ""
```

Comment:
Do you need to pass those parameters ?

Reply:
These parameters are marked as required in `template-e2e-test`:

https://github.com/kubeflow/katib/blob/51b246fa1c23db130c373fa9749897a0d0b334ab/.github/workflows/template-e2e-test/action.yaml#L6-L15

Maybe we can mark these parameters as not required. WDYT👀 @andreyvelich @tenzen-y

Reply:
Yeah, I think we can change them.

Reply:
Okay, I'll change them.